e2e-tests/pages/insurance/policy-and-export/aboutGoodsOrServices.js

```diff
@@ -25,7 +25,9 @@ const aboutGoodsOrServices = {
   },
   [FINAL_DESTINATION]: {
     label: () => cy.get(`[data-cy="${FINAL_DESTINATION}-label"]`),
-    input: () => cy.get(`[data-cy="${FINAL_DESTINATION}-input"]`),
+    input: () => cy.get(`[id="${FINAL_DESTINATION}"]`),
+    results: () => cy.get(`#${FINAL_DESTINATION} + ul li`),
+    noResults: () => cy.get('.autocomplete__option--no-results'),
```

Comment:
Could we have a shared field maybe in `pages/shared`? Otherwise we have the same selectors in multiple files just with a different ID being passed